compiler/rustc_typeck/src/coherence/builtin.rs

```diff
@@ -108,7 +108,7 @@ fn visit_implementation_of_copy(tcx: TyCtxt<'_>, impl_did: LocalDefId) {
     }
 }
 
-fn visit_implementation_of_coerce_unsized(tcx: TyCtxt<'tcx>, impl_did: LocalDefId) {
+fn visit_implementation_of_coerce_unsized(tcx: TyCtxt<'_>, impl_did: LocalDefId) {
```

Comment:
Maybe name `'tcx` here

Reply:
I tried to mostly follow the conventions of surrounding functions, but the rest of the file, other than this function and the one who follows use `'tcx'`, so I've changed both to follow the rest of the file.